src/components/DeeplinkWrapper/DeeplinkRedirectLoadingIndicator.tsx

```diff
@@ -12,23 +11,17 @@ import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import Navigation from '@libs/Navigation/Navigation';
 import ONYXKEYS from '@src/ONYXKEYS';
-import type * as OnyxTypes from '@src/types/onyx';
 
-type DeeplinkRedirectLoadingIndicatorOnyxProps = {
-    /** Current user session */
-    session: OnyxEntry<OnyxTypes.Session>;
-};
-
-type DeeplinkRedirectLoadingIndicatorProps = DeeplinkRedirectLoadingIndicatorOnyxProps & {
+type DeeplinkRedirectLoadingIndicatorProps = {
     /** Opens the link in the browser */
     openLinkInBrowser: (value: boolean) => void;
 };
 
-function DeeplinkRedirectLoadingIndicator({openLinkInBrowser, session}: DeeplinkRedirectLoadingIndicatorProps) {
+function DeeplinkRedirectLoadingIndicator({openLinkInBrowser}: DeeplinkRedirectLoadingIndicatorProps) {
     const {translate} = useLocalize();
     const theme = useTheme();
     const styles = useThemeStyles();
-
+    const [session] = useOnyx(ONYXKEYS.SESSION);
```

Comment:
We're only using `email`, so let's add a selector:

```suggestion
    const [email] = useOnyx(ONYXKEYS.SESSION, {selector: (session) => session?.email ?? ''});
```